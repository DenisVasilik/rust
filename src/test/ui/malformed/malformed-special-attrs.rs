#[cfg_attr] //~ ERROR expected `(`, found `<eof>`
struct S1;

#[cfg_attr = ""] //~ ERROR expected `(`, found `=`
struct S2;

#[derive] //~ ERROR attribute must be of the form `#[derive(...)]`
struct S3;

#[derive = ""] //~ ERROR attribute must be of the form `#[derive(...)]`
struct S4;

fn main() {}
