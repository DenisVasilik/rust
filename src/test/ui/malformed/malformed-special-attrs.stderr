error: expected `(`, found `<eof>`

error: expected `(`, found `=`
  --> $DIR/malformed-special-attrs.rs:4:12
   |
LL | #[cfg_attr] //~ ERROR expected `(`, found `<eof>`
   | - expected `(`
...
LL | #[cfg_attr = ""] //~ ERROR expected `(`, found `=`
   |            ^ unexpected token

error: attribute must be of the form `#[derive(...)]`
  --> $DIR/malformed-special-attrs.rs:7:1
   |
LL | #[derive] //~ ERROR attribute must be of the form `#[derive(...)]`
   | ^^^^^^^^^

error: attribute must be of the form `#[derive(...)]`
  --> $DIR/malformed-special-attrs.rs:10:1
   |
LL | #[derive = ""] //~ ERROR attribute must be of the form `#[derive(...)]`
   | ^^^^^^^^^^^^^^

error: aborting due to 4 previous errors

